tests/Integration/features/publicPageShare.feature

```diff
@@ -0,0 +1,65 @@
+Feature: publicPageShare
+
+  Scenario: Create and share a single page publicly (read-only)
+    When user "jane" creates collective "BehatPublicPageCollective"
+    And user "jane" creates page "singlesharepage" with parentPath "Readme.md" in "BehatPublicPageCollective"
+    And user "jane" creates public page share for page "singlesharepage" in "BehatPublicPageCollective"
+    Then anonymous sees public page share "singlesharepage" in collective "BehatPublicPageCollective" with owner "jane"
```

Comment:
Maybe check if anonymous cannot see the top level "Readme.md"?

Reply:
I'll add a check the anonymous cannot see `singlesharepage.md` in the share.